### PR TITLE
1.1.0 changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Relative links
- Release notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Release Information
 
-- **Version**:  1.0.1-1006
+- **Version**:  1.1.0-1003
```
Comment (cs-suyog-jinturkar) [inline README.md:3]: Please make this version consistent with other SPs

File: release_notes.md [hunk 2]
```diff
@@ -1,26 +1,8 @@
...
 
-The STIX endpoint to fetch datasets from a FortiSOAR instance has been enhanced to provide the list of feeds in a CSV format as well. With this change, systems such as FortiGate can directly pull threat intel via a FortiSOAR dataset.
+- With FortiSOAR v7.3.0 and FortiGuard Connector v1.1.0, the solution pack now enables support for Threat Intel Management SKU.
+    - Enable the Threat Intel Management SKU for best-in-class FortiGuard's threat intel feeds.
+    - The feeds from Threat Intel Management SKU have a rich variety of data, ranging from IPs, URLs, Domain and malicious hashes carefully curated by our team of experts.
+    - The entire feeds database is labelled with the relevant threat types, and associated LockHeed Martin Kill Chain Phases, that enables user with a lot of contextual information to understand the nature of threat.
 
-## Threat Intel Feed Enhancements
-
-The **Threat Intel Feeds** module better tracks the date and time at which the feeds were ingested or updated into the FortiSOAR instance.
```
Comment (cs-suyog-jinturkar) [inline release_notes.md:7]: Removed content is a good information, whether its also a part of Usage or content md file?

File: release_notes.md [hunk 3]
```diff
@@ -1,26 +1,8 @@
...
-## Dashboard Enhancements
-
-- All default dashboard components, like *Feeds Ingested*, now reflect statistics for Threat Intel Feeds ingested in the *Last 15 days*
-- *Active Feeds* is now renamed to *Ingestion Volume*
-
-## Bugfixes
-
-- **Get Total Item Count**, under **Threat Intel Management** > **Threat Feeds**, now displays the item count when clicked
-- The query to get Linked Indicators now filters the relevant linked indicators
+- With added support for the FortiGuard Feeds, the **Preview** label has been removed from the Solution Pack.
```
Comment (cs-suyog-jinturkar) [inline release_notes.md:8]: What does it mean for the customer?

File: README.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply README.md:3]: This is how it has always been for TIM.

File: release_notes.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply release_notes.md:7]: Yes, it is part of `usage.md`. Not in these exact same words, though.

File: release_notes.md (hunk 3 above)
Comment (cs-abhishek-shukla) [reply release_notes.md:8]: The solution pack's tile had the label "Preview". It is no longer there.

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**